apps/native-component-list/src/screens/Image/ImagePlaceholderScreen.tsx

```diff
@@ -0,0 +1,79 @@
+import { Image, ImageSource } from 'expo-image';
+import { useCallback, useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+import Button from '../../components/Button';
+import { Colors } from '../../constants';
+
+const generateSeed = () => 1 + Math.round(Math.random() * 2137);
```

Comment:
It's too much, let's not take up disk space unnecessarily
```suggestion
const generateSeed = () => 1 + Math.round(Math.random() * 10);
```